core/src/main/java/org/wso2/msf4j/util/Utils.java

```diff
@@ -114,4 +114,38 @@ public static int getIntersection(Set set1, Set set2) {
         cloneSet2.retainAll(cloneSet1);
         return cloneSet2.size();
     }
+
+    /**
+     * Get the path without leading or trailing / and with multiple / reduced to one /
```

Comment:
Shall we add single quotes for the slashes (ex: '/') since it will be more readable? WDYT?

Reply:
Will do that.